Trackovid19-web/src/app/app.component.ts

```diff
@@ -33,9 +33,7 @@ export class AppComponent {
     //TODO: Check if we want to go with this or not
     if (this.swUpdate.isEnabled) {
       this.swUpdate.available.subscribe(() => {
-        if (confirm('New version available. Load New Version?')) {
-          window.location.reload();
-        }
+        window.location.reload();
```

Comment:
Isn't this against the good practises?

Reply:
It's not the best UX but it's what product wants 😉and since the `skipWaiting` method is already called on the background, I don't see a reason to not update the page as soon as possible.

Reply:
Yes, it is. But this is not an ordinary application. Elder people will get confused and hesitant in answering that question. This way we get around this issue.